examples/gallery/textures/webpack.config.js

```diff
@@ -0,0 +1,19 @@
+const {resolve} = require('path');
+
+const CONFIG = {
+  mode: 'development',
+
+  entry: {
+    app: resolve('./app.js')
+  },
+
+  node: {
+    fs: 'empty'
+  },
+  devServer: {
+    contentBase: [resolve('.'), resolve('../../'), resolve('../../../')]
```

Comment:
I would prefer not doing this. Examples are intended to be standalone.

Reply:
removed